dataland-frontend/src/components/forms/parts/elements/basic/DateFormElement.vue

```diff
@@ -0,0 +1,96 @@
+<template>
+  <div class="lg:col-4 md:col-6 col-12 pl-0">
+    <Calendar
+      inputId="icon"
+      v-model="date"
+      :showIcon="true"
+      dateFormat="D, M dd, yy"
+      :maxDate="todayAsMax ? new Date() : undefined"
+      :placeholder="placeholder"
+    />
+  </div>
+
+  <FormKit
+    type="text"
+    :validation-label="validationLabel"
+    :validation="validation!"
```

Comment:
why is there a single `!`

Reply:
seemed to be not necessary. 
removed it and also similar cases in the other FormElements and FormFields